### ISSUE TITLE
difference between R-Drop and SimCse + Smart

### ISSUE THREAD

**Issue by cuixuage** [dropreg/R-Drop#21]
Appreciate for your work.  
why not try to smooth the embedding weights ?

**Comment by apeterswu** [comment]
The motivation behind these works is different, and also the specific operations. R-Drop investigates the dropout itself, i.e., the training and inference mismatch of model utilization. SimCSE and Smart focus on pre-training with robust embedding and incorporate adversarial training and contrastive learning. 

We do not aim at the embedding weights, as we claimed about the model differences. But of course, we can add to smooth the embedding weights.
